docs/08-tutorial.md

```diff
@@ -0,0 +1,94 @@
+## Tutorial
+In this tutorial, you create `TestDefinition` and `ClusterTestSuite` to show Octopus in action.
+
+1. Prepare data
+Create `TestDefinition` and `ClusterTestSuite`:
+
+```bash
+cd $GOPATH/src/github.com/kyma-incubator/octopus
+kubectl apply -f config/samples/testdefinition.yaml
+kubectl apply -f config/samples/testsuite.yaml
+
+```
+Created `TestDefinition` is very simple, it just executes `pwd` command in the `alpine:latest` 
```

Comment:
Not sure whether this sentence is necessary.

Reply:
removed